microprofile/messaging/core/src/test/java/io/helidon/microprofile/messaging/connector/ConnectorTest.java

```diff
@@ -19,17 +19,18 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
-import javax.enterprise.inject.spi.DeploymentException;
-import javax.enterprise.util.AnnotationLiteral;
-
 import io.helidon.microprofile.messaging.AbstractCDITest;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
+import jakarta.enterprise.inject.spi.DeploymentException;
+import jakarta.enterprise.util.AnnotationLiteral;
 import org.eclipse.microprofile.reactive.messaging.spi.Connector;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Disabled
```

Comment:
TODO: 3.0.0 JAKARTA

Reply:
Could you kindly write this one down? I would prefer that to adding another commit right now.